### PR TITLE
Removing restructuring leftover files

### PR DESCRIPTION
**Pull request by sodic**
These files aren't used anywhere but slipped through our restructuring process.

Verified file foo.ts wasn't used by running:
```
pwd && rg 'from.*foo' --no-filename -g '!sdk' | rg -v 'wasp'
```
Explanation:
 - `pwd` - prints the current directory to verify I'm in `templates`
 - `'from.*foo'` - finds all lines matching this regex (end of import statement)
 - `--no-filename` -excludes the filename from the output
 - `-g '!sdk'` - excludes SDK templates from the output (I only cleaned up the framework code)
 - `rg -v 'wasp'` - filters out all lines that include `wasp`, leaving only lines that both mention `foo` and don't mention the sdk package `wasp`.
 
If the final output is empty, it means that all imports mentioning `foo` are package imports coming from the SDK.

### CODE REVIEW

File: waspc/data/Generator/templates/react-app/src/entities/index.ts
Comment (sodic) [inline waspc/data/Generator/templates/react-app/src/entities/index.ts:1]: Only the SDK version is used. Proof:
```
$  pwd && rg 'import.*entities' --no-filename -g '!sdk'| rg -v 'wasp'
/home/filip/Projects/wasp/wasp-repo/waspc/data/Generator/templates
$
```

File: waspc/data/Generator/templates/react-app/src/queryClient.js
Comment (sodic) [inline waspc/data/Generator/templates/react-app/src/queryClient.js:1]: Proof:
```
$ pwd && rg 'import.*queryClient' --no-filename -g '!sdk' | rg -v 'wasp'
/home/filip/Projects/wasp/wasp-repo/waspc/data/Generator/templates
$
```

File: waspc/data/Generator/templates/react-app/src/storage.ts
Comment (sodic) [inline waspc/data/Generator/templates/react-app/src/storage.ts:1]: Proof:
```
$ pwd && rg 'import.*storage' --no-filename -g '!sdk' | rg -v 'wasp'
/home/filip/Projects/wasp/wasp-repo/waspc/data/Generator/templates
$
```

File: waspc/data/Generator/templates/server/src/actions/types.ts
Comment (sodic) [inline waspc/data/Generator/templates/server/src/actions/types.ts:1]: Proof:
```
$ pwd && rg 'import.*types' --no-filename -g '!sdk' | rg -v 'wasp'
/home/filip/Projects/wasp/wasp-repo/waspc/data/Generator/templates
$
```

File: waspc/data/Generator/templates/server/src/auth/utils.ts
Comment (sodic) [inline waspc/data/Generator/templates/server/src/auth/utils.ts:1]: Proof:
```
$ pwd && rg 'import.*utils' --no-filename -g '!sdk' | rg -v 'wasp'
/home/filip/Projects/wasp/wasp-repo/waspc/data/Generator/templates
$

```

File: waspc/data/Generator/templates/server/src/queries/types.ts
Comment (sodic) [inline waspc/data/Generator/templates/server/src/queries/types.ts:1]: Proof:
```
$ pwd && rg 'import.*types' --no-filename -g '!sdk' | rg -v 'wasp'
/home/filip/Projects/wasp/wasp-repo/waspc/data/Generator/templates
$
```

File: waspc/data/Generator/templates/server/src/utils.ts
Comment (sodic) [inline waspc/data/Generator/templates/server/src/utils.ts:1]: Proof:
```
$ pwd && rg 'import.*utils' --no-filename -g '!sdk' | rg -v 'wasp'
/home/filip/Projects/wasp/wasp-repo/waspc/data/Generator/templates
$
```

**Review by infomiho — APPROVED** [review]
Such a beautiful PR

**Pull request merged**